### PR TITLE
Addfields

### PR DESCRIPTION
**Pull request by sir-indy**
Added fields:
* description
* url
* cookTime
* prepTime

As discussed in #23

### CODE REVIEW

**Comment by sir-indy** [comment]
I didn't know that translation was a requirement. How do I correct accordingly?

File: templates/content/recipe.php [hunk 1]
```diff
@@ -6,9 +6,48 @@
...
+
+    <div class="recipe-details">
+        <p><?php echo $_['description']; ?></p>
+
+        <?php if(isset($_['prepTime']) && $_['prepTime']) {
+            $prep_interval = new DateInterval($_['prepTime']);
+            $prep_mins = $prep_interval->format('%i');
+            $prep_hours = $prep_interval->format('%h');
+            if($prep_hours > 0) {
+                $prep_hour_string = $prep_hours.' '.($prep_hours > 1 ? 'Hours' : 'Hour').' ';
```
Comment (christianlupus) [inline templates/content/recipe.php]:
```suggestion
                $prep_hour_string = $l->n('1 Hour','%n Hours', $prep_hours) . ' ';
```

File: templates/content/recipe.php [hunk 2]
```diff
@@ -6,9 +6,48 @@
...
+            } else {
+                $prep_hour_string = "";
+            }
+            if($prep_mins > 0) {
+                $prep_min_string = $prep_mins.' '.($prep_mins > 1 ? 'Mins' : 'Min');
```
Comment (christianlupus) [inline templates/content/recipe.php]:
```suggestion
                $prep_min_string = $l->n('1 Min', '%n Mins', $prep_mins);
```

File: templates/content/recipe.php [hunk 3]
```diff
@@ -6,9 +6,48 @@
...
+            } else {
+                $prep_min_string = "";
+            }
+            ?>
+            <p><strong>Prep: </strong><?php echo $prep_hour_string.$prep_min_string; ?></p>
```
Comment (christianlupus) [inline templates/content/recipe.php]:
```suggestion
            <p><strong><?php p($l->t('Prep')); ?>: </strong><?php echo $prep_hour_string.$prep_min_string; ?></p>
```

File: templates/content/recipe.php [hunk 4]
```diff
@@ -6,9 +6,48 @@
...
+        <?php } ?>
+        <?php if(isset($_['cookTime']) && $_['cookTime']) {
+            $cook_interval = new DateInterval($_['cookTime']);
+            $cook_mins = $cook_interval->format('%i');
+            $cook_hours = $cook_interval->format('%h');
+            if($cook_hours > 0) {
+                $cook_hour_string = $cook_hours.' '.($cook_hours > 1 ? 'Hours' : 'Hour').' ';
```
Comment (christianlupus) [inline templates/content/recipe.php]:
```suggestion
                $cook_hour_string = $l->n('1 Hour', '%n Hours', $cook_hours) . ' ';
```

File: templates/content/recipe.php [hunk 5]
```diff
@@ -6,9 +6,48 @@
...
+            } else {
+                $cook_hour_string = "";
+            }
+            if($cook_mins > 0) {
+                $cook_min_string = $cook_mins.' '.($cook_mins > 1 ? 'Mins' : 'Min');
```
Comment (christianlupus) [inline templates/content/recipe.php]:
```suggestion
                $cook_min_string = $l->n('1 Min', '%n Mins', $cook_mins);
```

File: templates/content/recipe.php [hunk 6]
```diff
@@ -6,9 +6,48 @@
...
+            } else {
+                $cook_min_string = "";
+            }
+            ?>
+            <p><strong>Cook: </strong><?php echo $cook_hour_string.$cook_min_string; ?></p>
```
Comment (christianlupus) [inline templates/content/recipe.php]:
```suggestion
            <p><strong><?php p($l->t('Cook')); ?>: </strong><?php echo $cook_hour_string.$cook_min_string; ?></p>
```

**Comment by sir-indy** [comment]
@mrzapp Thank you for merging this in, it's nice to be able to contribute even in a small way, I hope other people find it useful!
@christianlupus Thank you for your help with adding in the translation tags, I've not used PHP before so didn't know to use them. I'll make sure I do it like you showed me next time.

**Pull request merged**